bundles/org.openhab.binding.weatherunderground/src/main/resources/OH-INF/thing/bridge.xml

```diff
@@ -7,7 +7,7 @@
 	<bridge-type id="bridge">
 		<label>Weatherunderground Bridge</label>
 		<description>The Weather Underground bridge represents a connection to the Weather Underground service</description>
-
+		<semantic-equipment-tag>NetworkAppliance</semantic-equipment-tag>
```

Comment:
There is something inconsistent.
You use NetworkAppliance for this weather service while WebService was used for the openweathermap binding.

And same question than for the openweathermap binding, should't we use WeatherService instead?

Reply:
If the thing is associated with a physical local piece of equipment attached to the lan it is a network appliance. If thing is attached directly to a remote cloud based service it is a web service. And if it is a specific local widget that measures (say) temperature, humidity, barometric pressure, it is a weather station. Of course it may be that the thing may have attributes of all three of the above; in which case we need to decide which of those three is the primary and best description of that thing.

Reply:
> If thing is attached directly to a remote cloud based service it is a web service.

So it should be WebService here as there is no physical device, just an attachment to a remote cloud based service.